src/NServiceBus.Ninject/obsoletes.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace NServiceBus
+{
+    /// <summary>
+    /// Ninject Container
+    /// </summary>
+    [ObsoleteEx(Message = "NServiceBus.Ninject is no longer supported. Use the externally managed container mode to integrate third party dependency injection containers with NServiceBus instead.",
+        TreatAsErrorFromVersion = "8",
+        RemoveInVersion = "9")]
+    public class NinjectBuilder
+    {
+    }
+
+    /// <summary>
+    /// Ninject extension to pass an existing Ninject container instance.
+    /// </summary>
+    [ObsoleteEx(Message = "NServiceBus.Ninject is no longer supported. Use the externally managed container mode to integrate third party dependency injection containers with NServiceBus instead.",
+        TreatAsErrorFromVersion = "8",
+        RemoveInVersion = "9")]
+    public static class NinjectExtensions
+    {
+    }
+
+}
+
+namespace NServiceBus.ObjectBuilder.Ninject
+{
+    using System;
+    using global::Ninject.Syntax;
+
+    /// <summary>
+    /// Contains extension methods to configure UnitOfWork scoped bindings.
+    /// </summary>
+    [ObsoleteEx(Message = "NServiceBus.Ninject is no longer supported. Use the externally managed container mode to integrate third party dependency injection containers with NServiceBus instead.",
```

Comment:
Are these method obsolete necessary when we obsolete the whole class as well?

Reply:
yes because when you use the extension method, the obsolete from the class type isn't showing up. But when you do `NinjectObjectBuilderExtensinos.SomeExtensions(...)` it does